src/App.tsx

```diff
@@ -1,7 +1,35 @@
 import React from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+type State = {
+  key: string,
+};
+// eslint-disable-next-line react/prefer-stateless-function
+export class App extends React.Component {
```

Comment:
You should provide the props argument to the React.Component, even if it's empty. Change to 'React.Component<{}, State>'.